### PR TITLE
Fix project.yml

### LINKED ISSUE #217: eRouška in XcodeGen/Docs/Examples.md
Hello.
Thank you for erouska-ios.
I just want to notify you that I added eRouška to [XcodeGen/Docs/Examples.md](https://github.com/yonaskolb/XcodeGen/blob/master/Docs/Examples.md).

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for erouska-ios.
Fixed #217.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 
-bundle install
```
Comment (RomanPodymov) [inline setup.sh:3]: There is no Gemfile in the project.

File: project.yml [hunk 2]
```diff
@@ -76,8 +76,8 @@ targets:
         com.apple.security.network.server: true
         com.apple.developer.exposure-notification: true
     configFiles:
-      Debug: Configs/Prod-signing.xcconfig
-      Release: Configs/Prod-signing.xcconfig
+      Debug: Configs/Templates/Prod-signing.xcconfig
```
Comment (RomanPodymov) [inline project.yml]: Just fix the paths.

Comment (augard) [reply project.yml]: Point of it it's to copy template and create your own. Becouse you need to set your own provision profiles. So you will be able to run it on device.

**Review by augard — CHANGES_REQUESTED** [review]
Please remove Templates folder from paths

File: project.yml (hunk 2 above)
Comment (augard) [reply project.yml]: from readme:

**Code signing**
To update your code signing settings, you would need to copy template .xcconfig files from Configs/Templates/ directory to Configs/ directory and fill them with proper values. xcodegen is looking for these files in Configs directory and use them in project generation. All .xcconfig files in Configs directory are ignored by git, so you don't have to worry about accidentally pushing your code signing settings or pulling somebody elses.

Comment (RomanPodymov) [reply project.yml]: Hello @augard 
Sorry for the delay. My suggestion is to define configs path as a constant, for example using `export EROUSKA_CONFIGS_PATH="My/Path/For/Configs"` (maybe with the default value "Configs/Templates"). If you agree with that, I can also fix README.md. If not, I will just revert the changes I did in project.yml.

**Comment by RomanPodymov** [comment]
> Please remove Templates folder from paths

Fixed.

**Pull request closed without merging**